sys-fs/mount-zip/mount-zip-1.0.9.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="FUSE file system for ZIP archives"
+HOMEPAGE="https://github.com/google/mount-zip"
+SRC_URI="https://github.com/google/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+
+RDEPEND="
+	dev-libs/icu
```

Comment:
Shouldn't this use the `=` slot-operator for icu and libzip? So, e.g., `dev-libs/icu:=`.

Reply:
Done